DeAceroDLabs/marketplace-frontend
# Feature/dmp 460

## What is done

[[DMP-460] Carga valores usando el código postal](https://deacero.atlassian.net/browse/DMP-460)

## How is done

1. On Signup used the watch function to check for change on the zipCode fields.
2. Added the new location API method.
3. Created a new context for storing the dynamic data. 
4. Modified the Dropdown component to get the options from the new context if available.

## How to test

1. Run `npm start`.
2. Go to `/signup`.
3. Go to the second form, location.
4. On the "código postal" field, add a zipCode available on México, at least 5 digits.
5. The City and State fields should get the values from the locations service once a zipCode is introduced.
6. The "Colonias" field should be filled with the options for that specific zipCode.
